Source/DafnyLanguageServer.Test/Various/CompilationStatusNotificationTest.cs

```diff
@@ -97,10 +97,6 @@ method Abs(x: int) returns (y: int)
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
       await AssertProgress(documentItem, CompilationStatus.ResolutionStarted);
       await AssertProgress(documentItem, CompilationStatus.CompilationSucceeded);
-      await AssertProgress(documentItem, CompilationStatus.VerificationStarted);
-      await AssertProgress(documentItem, CompilationStatus.VerificationStarted, "0/1 Abs");
-      await AssertProgress(documentItem, CompilationStatus.VerificationStarted, "1/1 (Abs finished)");
-      await AssertProgress(documentItem, CompilationStatus.VerificationSucceeded);
```

Comment:
What happened to this test? Just wanted to confirm that you disabled this behavior so we are not left with untested code.

Reply:
I removed the code that emits VerificationStarted and VerificationSucceeded, so there wasn't anything left to test.

Reply:
The behavior was removed, so there was nothing left to test.